app/src/main/AndroidManifest.xml

```diff
@@ -36,6 +36,12 @@
 
     <supports-screens android:largeScreens="true" />
 
+    <queries>
```

Comment:
https://developer.android.com/training/package-visibility

So we have the correct messages packs installed